README.md

```diff
@@ -1,2 +1,45 @@
-# ams-operator
- Anbox Management Service
+# AMS Operator
+
+Anbox Cloud offers a software stack that runs Android applications in any cloud enabling high-performance
+streaming of graphics to desktop and mobile client devices.
+
+At its heart, it uses lightweight container technology instead of full virtual machines to achieve higher
+density and better performance per host while ensuring security and isolation of each container. Depending
+on the target platform, payload, and desired application performance (e.g. frame rate), more than
+100 containers can be run on a single machine.
+
+For containerization of Android, Anbox Cloud uses the well established and secure container hypervisor
+LXD. LXD is secure by design, scales to a large number of containers and provides advanced resource
+management for hosted containers.
+
+Also have a look at the official Anbox Cloud website (https://anbox-cloud.io) for more information.
+
+> NOTE: Anbox Cloud is a paid offering. You will need a Ubuntu Pro (https://ubuntu.com/pro) subscription
+> for this charm to work. You can learn more at https://anbox-cloud.io
+
+> **WARNING:** The *Ubuntu Pro (infra only)* subscription every Ubuntu user gets for free for
+> personal use does **NOT** work and will result in a failed deployment! You either need a
+> *Ubuntu Pro* or *Ubuntu Pro (apps only)* subscription to be able to deploy successfully.
+
+## Anbox Management System
```

Comment:
This is just a copy of the documentation for the current ops charm. This will be re done anyway in the upcoming few commits to be in line with the documentation patterns for the other charms.

Reply:
That's fine but we should always consistency between what the implementation does and what we have in the README. Otherwise it's too easy to miss changing later or other things come in between.

Reply:
This still needs fixing.

Reply:
This stands fair now since I have added the etcd relation to the charm as well?